contracts/src/factory/facets/create/CreateSpace.sol

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+// interfaces
+import {ICreateSpace} from "contracts/src/factory/facets/create/ICreateSpace.sol";
+
+// libraries
+
+// contracts
+import {Facet} from "contracts/src/diamond/facets/Facet.sol";
+import {ArchitectBase} from "contracts/src/factory/facets/architect/ArchitectBase.sol";
+import {PausableBase} from "contracts/src/diamond/facets/pausable/PausableBase.sol";
+import {ReentrancyGuard} from "contracts/src/diamond/facets/reentrancy/ReentrancyGuard.sol";
+
+contract CreateSpaceFacet is
+  ICreateSpace,
+  ArchitectBase,
+  PausableBase,
```

Comment:
What bothers me is all facets on a diamond use the same `PausableBase` lock while we may only want to pause certain facets.

Reply:
Good point, let's discuss when we encounter this issue.